_data/ball/2023.yaml

```diff
@@ -0,0 +1,58 @@
+name: CSS BALL
+year: 2023
+location: Fazeley Studios
+date: 26th April
+links: 
+  - text: Menu
+    url:  '/ball/2023/menu'
+  - text: Tickets
+    url:  '#tickets'
+  - text: FAQs
+    url: '#faq'
+menu-links:
+  - text: Home
+    url:  '/ball/2023'
+  - text: Tickets
+    url:  '/ball/2023#tickets'
+  - text: FAQs
+    url:  '/ball/2023#faq'
+menu:
+  starters:
+    - dish: Ham hock & Cumberland sausage roulade, sweet potato spring onion salsa & sage crisps
+      type: dairy free
+    - dish: Pan fried chilli polenta, roasted ginger beets, samphire with an doragen and coriander dressing
+      type: vegan & gluten free
+  mains:
+    - dish: Pan seared thyme and garlic chicken, creamy dauphinoise potato, sautéed green beans, baby carrots, and thyme chicken jus
+      type: gluten free
+    - dish: Chargrilled sweet potato dome filled with ratatouille, courgette, adn carrot crisps, and chive cream
+      type: vegan & gluten free
+  dessert:
+    - dish: Belgian dark chocolate mousse with caramelised celementine orange and deep-fried mint
+      type: vegan & gluten free
+  
+faq:
+  - question: What is a Ball?
+    answer: A ball is a solid or hollow spherical or egg-shaped object that is kicked, thrown, or hit in a game. <br><br> Ok but seriously, a Ball is an opportunity to come together and celebrate the year. Many societies hold them for their members and we're no different. Our annual Ball is the biggest night of the year so if you're not attending you're really missing out.
+  - question: What will be there?
+    answer: The ball will have a 3 course meal, a bar, and a dancefloor. We will also have a live band, a photobooth, and a photographer.
+  - question: When can I buy tickets?
+    answer: Now! You'll need to be a CSS Member to purchase tickets - if you aren't then you can join CSS on the <a href="/join">Guild website</a>.
+  - question: How much do tickets cost?
+    answer: Tickets are £42.50, same as last years. The Guild will also add on a booking fee - we can't do anything about that.
+  - question: Can I bring a guest?
+    answer: Yes, you will need to buy an extra ticket for your guest.
```

Comment:
Can we write a slightly passive aggressive line in here saying that the Guild haven't enabled this on the website at the moment but we will be fixing this as soon as we can?

Reply:
oh honey I have _so_ much more where that came from. Enjoy this game of 'spot the difference' between what we requested and what the Guild put up originally.
![cursed_ball_tickets](https://user-images.githubusercontent.com/9078727/229860181-446b915a-d6d6-43bf-8780-a09910e42ef8.png)

Reply:
🤡

Reply:
Oh another one @codemicro - there isn't enough room to fit 3 songs into the song request box: 
![image](https://user-images.githubusercontent.com/9078727/229864802-5c70a637-6aad-4c2b-8b14-4056fefb16e0.png)